tools/mkpkg/update_retroplayer-addons

```diff
@@ -163,9 +163,7 @@ for addon in ${ADDONS_DIR}/*.*/ ; do
     else
       msg_warn "unmanaged version in kodi package: ${VERSION_INFO}"
       # unmanaged version, repo plus branch
-      RETRO_SITE=$(echo "${VERSION_INFO}" | awk '{print $2}')
-      RETRO_BRANCH=$(echo "${VERSION_INFO}" | awk '{print $3}')
-      RETRO_NEW_VERSION=$(git ls-remote "${RETRO_SITE}" "${RETRO_BRANCH}" | awk '{print $1}')
+      RETRO_NEW_VERSION=$(echo "${VERSION_INFO}" | sed "s|.*/archive/\(.*\)\.tar.gz\.*|\1|")
```

Comment:
Please add tar.gz handling as a separate elif branch, like zip handling above, eg like this: https://github.com/HiassofT/LibreELEC.tv/commit/5299a943d3274ac1bf87a2a3847f5ba34dea2d24

Not 100% sure if zip and unmanaged branch handling are still needed, so better keep them in for now, we can drop that later

Reply:
Will try to implement it!

This retro games script looks outdated and unused. Even some retro packages do have another source URL than the XBMC repo txt file does point to. So this looks outdated for me as well.

I only touched it because to check if the API is working.

Reply:
thanks!

You can leave that out for now and do it separately, there seem to be some more issues (like -Leia tags getting picked up as Leia/Matrix/Nexus tags all point to the same hash and the `-KODI_BRANCH` match not working ATM as that's master, not Nexus...)